react/features/base/react/components/native/NavigateSectionListItem.js

```diff
@@ -100,6 +105,24 @@ export default class NavigateSectionListItem extends Component<Props> {
         return lines && lines.length ? lines.map(this._renderItemLine) : null;
     }
 
+    /**
+     * Renders the secondary action label.
+     *
+     * @private
+     * @returns {React$Node}
+     */
+    _renderSecondaryAction() {
```

Comment:
@virtuacoplenny has introduced elementAfter solution when implementing the web + button and I have used the same for adding Join. So, I think we should choose one and stick with it for mobile and web. I like the secondary action approach, but elementAfter is also very flexible, so basically we can put anything there. Let me know what your thoughts are on that and do you have any preference?

Reply:
Zoli wants the secondaryAction.`I really don't like the react components being pushed into the displayableItem in BaseCalendarList, because I intended that object to be pure data...` Considering he's the original implementer, and there is no use case yet for the flexibility of an elementAfter, just going with secondaryAction seems fine.

Reply:
@zbettenbuk Ok, I'll rework the web part once this pr is merged.

Reply:
I've had a closer look and currently the secondaryAction implementation is rather specific and is bound to the + button and its use case. We need to make this more generic, so that we can use it show another button (the "Join" button on web)

Reply:
We discussed this with Lenny, this seems to be okay this way. Mobile may need to adjust too, because we may have more actions in those lists as well